slave/url_creator.py

```diff
@@ -64,21 +64,53 @@ def latest(self, **kwargs):
 class TaskClusterIndexHelper(object):
     _index_url = 'https://index.taskcluster.net/v1/task/gecko.v2'
     _artifacts = 'https://public-artifacts.taskcluster.net'
+    _task_inspector = 'https://tools.taskcluster.net/tasks'
```

Comment:
I trust your knowledge of TaskCluster et al. here, because I don't have any, but that'd be nice that somebody with some TC knowledge would chime in.